### DISCUSSION TITLE
How do I do a fresh install of windows 11 for best computer performance and compatibility with g-helper? ASUS ROG Zephyrus G14 (2024)

### DISCUSSION THREAD

**Discussion by grantandrewsnc** [seerge/g-helper#2979] [Q&A]
### This post is obviously for me but also anyone with a new ASUS gaming laptop and anyone who wants to do a fresh windows 11 install for best performance and compatibility with g-helper.

1st paragraph is some background, 2nd are my problems, and 3rd is the TL;DR and my support question/ request.

### Background:
I just got a new **ASUS ROG Zephyrus G14 (2024) Running an AMD Ryzen 9 8945HS with a NVIDIA GeForce RTX 4070** a couple of days ago and have recently been setting it up and cleaning out bloat and trying to optimize it with g-helper. I got g-helper running and in the process of cleaning out applications I didn't want I deleted One Drive without disabling sync; which unfortunately has the side effect of deleting your desktop folder... Since that's happened, I've tried to fix it in multiple ways but haven't been able to get my files working like I want. Additionally, since that has happened g-helper also has been a little troublesome in that it doesn't start immediately unless I manually open the application every time I start my computer. Anyway, from what I've read it seems like a fresh windows 11 install also gets rid of a lot of bloatware on the computer and can help make sure that g-helper runs as intended without any conflict with My Asus or Armory Crate. Because this is a new computer, I want it running as smoothly as possible with as few extra applications that I don't use installed while also running g-helper in the most compatible way possible.

### Here are my PROBLEMS:
-- I can't seem to fix my Desktop folder problem after deleting One Drive even after trying multiple different ways to fix that. In the process of deleting bloat off of my computer, after I had uninstalled My Asus, Armoury Crat and installed g-helper, I think I may have left some remnants of My Asus and/or Armoury Crate on my computer because g-helper won't start right away without manually running it. I even used the Armoury Crate uninstall tool and have looked into all settings on g-helper and in windows. 
-- I didn't really want to **fully reinstall windows 11 on my computer and then reinstall all of the _necessary_ ASUS, AMD, and NVIDIA programs and drivers as well as g-helper** because it seems like a lot of things that I could mess up. I do mostly know what I am doing but don't want to mess something up in the process of a complete reinstall. 
-- After some research it seems like reinstalling windows 11 helps with getting rid of bloat and also improves performance and g-helper compatibility within ASUS computers which sounds ideal to me. This has convinced me that reinstalling windows 11 with everything else will fix all of my problems. Although, because I don't want to mess any of this process up, I need a full guide on how to do this whole reinstall of windows 11 and everything else. The problem is that even after extensive searching I can't find a full guide on this anywhere; Reddit, here, or the rest of the internet. 

### TL;DR and my Support Request Question:
TL;DR
I just got a new ASUS gaming computer and want to do a fresh install of windows 11 making sure that I also download all of the necessary AMD, ASUS, and NVIDIA drivers and software to make my computer run smoothly and have the best performance, cleanest and least bloated system, and best compatibility with g-helper. 

Support Request / Questions
Would it be possible to get **a comprehensive guide on how to do a fresh install of windows 11 including all of the ASUS, AMD, and NVIDIA drivers and software that I need to run my G14 smoothly and in the most compatible way with g-helper**? Are there any files that I need to download to a USB drive before I uninstall my current version of Windows 11 and reinstall it? 

Any help would be greatly appreciated!
(Also sorry for any redundancies in this post if you read the whole thing, I am just trying to make this as easy for people to read as possible if they only wanted to read some of it)

Thank YOU!
